src/net/ossl.cc

```diff
@@ -79,6 +79,8 @@ module seastar;
 #include "net/tls-impl.hh"
 #endif
 
+template <> struct fmt::formatter<seastar::tls::session> : fmt::ostream_formatter {};
```

Comment:
is it possible to implement the specialization of `fmt::fornmatter<seastar::tls::session>` without the `operator<<`? it would be better if we can avoid adding more `operator<<`, for two reasons:

1. avoid adding the unused `operator<<`
2. avoid the overhead of using a temporary ostream for formatting `seastar::tls::session`.